Sources/Server/Parser.swift

```diff
@@ -12,16 +12,17 @@ public class Parser {
     public func parseRequest(request: String) -> Request {
         var (requestLineComponents, requestHeaders, requestBody) = parseRequestMessage(request: request)
         let headers = parseHeaders(requestHeaders: requestHeaders)
-        if (requestLineComponents[1].contains("?")) {
-            let parsedPathWithQueries = parsePath(path: String(requestLineComponents[1]))
-            let queries = parseAllQueries(parsedQuery: parsedPathWithQueries[1])
-            return Request(directory: directory, method: String(requestLineComponents[0]), path: parsedPathWithQueries[0], httpVersion: String(requestLineComponents[2]), queries: queries, body: requestBody, headers: headers) 
-        } else if (headers.keys.contains("Cookie")) {
-            let cookie = parseCookie(headers: headers)
-            return Request(directory: directory, method: String(requestLineComponents[0]), path: String(requestLineComponents[1]), httpVersion: String(requestLineComponents[2]), body: requestBody, cookie: cookie["type"]!, headers: headers) 
```

Comment:
Ah I see. You were always assuming that the ["Type"] Cookie would be present when any cookie was present. Makes sense why this was breaking when deployed!